content/en/docs/unit-testing.md

```diff
@@ -5,11 +5,15 @@ description: "How to mock the AWS SDK for Go V2 when unit testing your applicati
 weight: 9
 ---
 
-You can mock out the AWS SDK for Go V2 when unit testing your application by
-using Go interfaces. Using interface definitions you define the set of
-operations required by your application, and provide mock implementations of
-this interface when unit testing. You can follow this pattern to unit testing
-service client operations, paginators, and waiters.
+When using the SDK in your application, you'll want to mock out the SDK for
+your application's unit test. Mocking out the SDK allows your test to be
+focused on what you want to test, not the internals of the SDK.
+
+To support this your application should use Go interfaces instead of the
+concrete service client, paginators, and waiter types, (e.g. `s3.Client`).
+Using a Go interface to specify the methods your application will use, allows
+your application to use patterns such as to dependency injection. Where the
+concrete implementation is passed in, based on the situation, such as testing.
```

Comment:
Curious on the wording and tone change?

Reply:
Main intention here was to focus on how user should use the SDK within their application. I don' think the original wording emphasized the how an application would need to be structured/designed in order to be able to mock out the application in it.